### PR TITLE
Contains 2 new tasks for creating dgeAnalysis input files

### PR DESCRIPTION
**Pull request by tomkuipers1402**
### Checklist
- [x] Pull request details were added to CHANGELOG.md
- [x] `parameter_meta` for each task is up to date.

### CODE REVIEW

File: prepareShiny.wdl [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        String dockerImage = "tomkuipers1402/shiny-py:v1.0.1"
+    }
+
+    command {
+        set -e
+        mkdir -p ${shinyDir}
+        sampleSheet.py \
+            -i ${countTable} \
+            -o ${shinyDir}
+        cp ${countTable} ${shinyDir}/allCounts.tsv
```
Comment (DavyCats) [inline prepareShiny.wdl:40]: Why is the count table getting copied?

File: prepareShiny.wdl [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        timeMinutes: {description: "The maximum amount of time the job will run in minutes.", category: "advanced"}
+        dockerImage: {description: "The docker image used for this task. Changing this may result in errors which the developers may choose not to address.",
+                      category: "advanced"}
+    }
+}
+
+task CreateAnnotation {
+    input {
+        File referenceFasta
+        File? referenceGtfFile
```
Comment (DavyCats) [inline prepareShiny.wdl:70]: This doesn't look it is optional, based on the command.

**Review by DavyCats — CHANGES_REQUESTED** [review]

File: prepareShiny.wdl (hunk 1 above)
Comment (tomkuipers1402) [reply prepareShiny.wdl:40]: To collect all input files in 1 directory, maybe a bit redunant indeed

File: prepareShiny.wdl (hunk 2 above)
Comment (tomkuipers1402) [reply prepareShiny.wdl:70]: Your right, I used the "?" based on the input list in RNA-seq.wdl. Can I remove the question mark here or should/can I remove the question mark in RNA-seq.wdl?

Comment (DavyCats) [reply prepareShiny.wdl:70]: It shouldn't be removed from RNAseq. Just remove it here and put the call to this task in an `if defined(referenceGtfFile)` block

File: prepareShiny.wdl [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task CreateSampleGetCount {
```
Comment (DavyCats) [inline prepareShiny.wdl:23]:
```suggestion
task CreateSamplesheet {
```

**Review by DavyCats — APPROVED** [review]

File: prepareShiny.wdl [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+task CreateDesignMatrix {
+    input {
+        File countTable
+        String shinyDir = "."
+
+        Int threads = 1
+        String memory = "5G"
+        Int timeMinutes = 30
+        String dockerImage = "tomkuipers1402/shiny-py:v1.0.2"
```
Comment (JasperBoom) [inline prepareShiny.wdl:31]: Maybe its a good idea to also put this image under the docker hub account of the LUMC?

**Review by JasperBoom — APPROVED** [review]

**Pull request closed without merging**